solutions/prob6.py

```diff
@@ -0,0 +1,34 @@
+# https://projecteuler.net/problem=6
+# Sum square difference of first 100 natural numbers
+
+import math
+
+class sumSquareDiff(object):
+	
+	def __init__(self, number):
+        	self.number = number
```

Comment:
Change the argument name `number` to `endrange` or something? `number` is kind of confusing.